receiver/fluentforwardreceiver/README.md

```diff
@@ -18,13 +18,25 @@ This receiver:
  - If using TCP, it will start a UDP server on the same port to deliver
    heartbeat echos, as per the spec.
 
+Fluentforward doesn't have a native notion of log body, severity, resources and so on.
+We could considere evething as structured body or attributes. Attributes has been choosed to ease processors manipulation.
+With `mappings` config you can move a fluent attribute to a body attribute or as severity text.
+In this case, all found attributes will be moved.
+
 Here is a basic example config that makes the receiver listen on all interfaces
 on port 8006:
 
 ```yaml
 receivers:
   fluentforward:
     endpoint: 0.0.0.0:8006
+    body_as_string: true
+    mappings:
```

Comment:
If anyone has a better idea / strategy about these mapping I take it. It makes the job but I have the feeling that it's not ideal.
But what I think is the concept could be reused on other receivers where underlying technology doesn't fit with all OTEL concepts.